src/components/actionPayloadRenderers/SessionPayloadRenderer.tsx

```diff
@@ -25,11 +25,14 @@ export default class Component extends React.Component<Props, {}> {
             ? null
             : sessionAction.renderValue(Util.createEntityMapFromMemories(entities, memories), { fallbackToOriginal: true })
 
-        return <div className={`${OF.FontClassNames.mediumPlus} cl-text-payload cl-font--emphasis`}>
-            <div className="cl-text-payload__string" data-testid="action-scorer-text-response">
-                EndSession: {(renderStringUsingCurrentMemory === null)
-                    ? `${renderStringUsingEntityNames}`
-                    : `${renderStringUsingCurrentMemory}`
+        return <div className={`${OF.FontClassNames.mediumPlus}`}>
+            <div className="cl-payloadrenderer-session-primary" data-testid="action-scorer-session-response">
```

Comment:
Both of these `data-testid` attributes are `action-scorer-session-response` they should be unique through application to guarantee tests select the right item.

If could put the `data-testid` attribute on the parent `<div>` to only have one. Or maybe separate into system and data responses

Reply:
Good catch. I've appended "-extended" to the second, optional DIV as that's extended session data.